platform/vs/docker-physyncd-vs/supervisord.conf

```diff
@@ -3,21 +3,39 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:start.sh]
-command=/usr/bin/start.sh
-priority=1
+[eventlistener:dependent-startup]
+command=python -m supervisord_dependent_startup
 autostart=true
-autorestart=false
+autorestart=unexpected
+startretries=0
+exitcodes=0,3
+events=PROCESS_STATE
+
+[eventlistener:supervisor-proc-exit-listener]
+command=/usr/bin/supervisor-proc-exit-listener --container-name physyncd
```

Comment:
Please take a close look at this container-name arg. I wasn't sure if it should be syncd (the name of the syncd binary is unchanged in physyncd) or what I chose here, physyncd.

Reply:
These changes have basically killed everything, going to back them out. Nothing starts (has negative in swss and physyncd).

Reply:
@jleveque I finally root caused my local test failures, the problem was I needed to pull in more of the changes from pull request #4599.  We should be good now, fundamentally the same supervisord setup used in master syncd now applies for physyncd in this pull request. Thanks for your patience and pointing me in the right direction.